src/logic/import/voc/VOCImporter.ts

```diff
@@ -0,0 +1,142 @@
+import {ImageData, LabelName, LabelRect} from '../../../store/labels/types';
+import {LabelUtil} from "../../../utils/LabelUtil";
+import {AnnotationImporter} from '../AnnotationImporter';
+import {LabelsSelector} from '../../../store/selectors/LabelsSelector';
+
+type FileParseResult = {
+    filename: string,
+    labeledBoxes: LabelRect[]
+};
+
+type VOCImportResult = {
+    labelNames: Record<string, LabelName>,
+    fileParseResults: FileParseResult[],
+};
+
+class DocumentParsingError extends Error {
```

Comment:
@SkalskiP These types of errors are likely similar to existing YOLO and COCO errors and could be genericized but since I'm only using them in this file at the moment just to indicate what imported file had the issue, I was planning to just keep them here for now. Let me know if you'd like me to move it out